src/common/translation/i18n/en.json

```diff
@@ -77,7 +77,7 @@
       }
     },
     "message": {
-      "eventVisitsThisYear": "You have attended events this year"
+      "eventVisitsThisYear": "Events you have registered this year"
```

Comment:
I think this translation might be better, maybe?

"Events you have registered *for* this year"